### ISSUE TITLE
No archives to choose from

### ISSUE THREAD

**Issue by ChillHorse** [Adamantcheese/Kuroba#492]
![Screenshot_20200106-143509](https://user-images.githubusercontent.com/17539404/71821319-1c841c80-3092-11ea-8193-8c1999cf9177.png)

Cannot select any archive to view old threads

**Comment by K1rakishou** [comment]
Duplicate of #480 will be fixed once #481 is merged.
